src/vm/wren_utils.h

```diff
@@ -64,6 +65,15 @@ typedef struct {
     void wren##name##BufferWrite(WrenVM* vm, name##Buffer* buffer, type data) \
     { \
       wren##name##BufferFill(vm, buffer, data, 1); \
+    } \
+    int wren##name##BufferFind(name##Buffer* buffer, type data) \
+    { \
+      for (int i = 0; i < buffer->count; i++) \
+      { \
+        if (memcmp(&buffer->data[i], &data, sizeof(data)) == 0) return i; \
+      } \
+      \
+      return -1; \
```

Comment:
Since this is only used in one place, at least right now, let's just make a little standalone static function in wren_compiler.c just for the use case we need. The Buffer stuff isn't really like an OOP class where every "method" that works on it needs to be part of it. It's fine to have unique functions that poke at buffers of specific kinds.